pkg/cmd/delete.go

```diff
@@ -0,0 +1,53 @@
+package cmd
+
+import (
+	"github.com/hobbyfarm/hfcli/pkg/scenario"
+	command "github.com/rancher/wrangler-cli"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+type Delete struct{}
+
+func NewDelete() *cobra.Command {
+	delete := command.Command(&Delete{}, cobra.Command{
+		Use:   "delete",
+		Short: "delete objects, valid options are scenario",
+	})
+	delete.AddCommand(
+		NewDeleteScenario(),
+	)
+	return delete
+}
+
+func (a *Delete) Run(cmd *cobra.Command, args []string) error {
+	if len(args) == 0 {
+		return cmd.Help()
+	}
+	return nil
+}
+
+type DeleteScenario struct{}
+
+func NewDeleteScenario() *cobra.Command {
+	deleteScenario := command.Command(&DeleteScenario{}, cobra.Command{
+		Use:   "scenario",
+		Short: "delete scenario NAME",
+	})
```

Comment:
might be better to leverage something like `Args: cobra.ExactArgs(1),` in the command definition to ensure that just a single argument is supplied. It will simplify the `Run` call.